laravel/app/Http/Controllers/SlotController.php

```diff
@@ -167,17 +168,16 @@ public function edit(SlotEditRequest $request, Slot $slot)
         $slot->save();
         return;
     }
+
     public function adminRelease(Request $request, Slot $slot)
     {
-        
         if(Auth::user()->hasRole('admin'))
         {
-            $username = $slot->user->data->burner_name;
+            $username = (!is_null($slot->user->data->burner_name) ? $slot->user->data->burner_name : $slot->user->data->full_name);
```

Comment:
This should use the same logic as the report controller instead of using `!is_null()`